<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trajectory to GIF</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnPlay">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>390</y>
      <width>82</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRecord">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Render all selected items as GIF</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lstSignals">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>531</width>
      <height>301</height>
     </rect>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
    <property name="uniformItemSizes">
     <bool>true</bool>
    </property>
    <property name="selectionRectVisible">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenLog">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>82</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Open log...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnChangeOutput">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Change output path...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtOutputPath">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinPlayback">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>48</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>750</number>
    </property>
    <property name="value">
     <number>150</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Frames per second</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPreview">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>40</y>
      <width>380</width>
      <height>380</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>Preview</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of trace lines</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinTrace">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>48</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>15</number>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Width of trace lines</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>20</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinWidth">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>48</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbExtension">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>gif</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mp4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>avi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mpg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mov</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mkv</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>wmv</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>355</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Output format</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinAnchorFrames">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>380</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string># of black frames before playback</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>btnPlay</zorder>
   <zorder>btnRecord</zorder>
   <zorder>lstSignals</zorder>
   <zorder>btnOpenLog</zorder>
   <zorder>btnChangeOutput</zorder>
   <zorder>txtOutputPath</zorder>
   <zorder>spinPlayback</zorder>
   <zorder>lblPreview</zorder>
   <zorder>label_3</zorder>
   <zorder>spinTrace</zorder>
   <zorder>line</zorder>
   <zorder>label_4</zorder>
   <zorder>line_2</zorder>
   <zorder>spinWidth</zorder>
   <zorder>cmbExtension</zorder>
   <zorder>label_2</zorder>
   <zorder>spinAnchorFrames</zorder>
   <zorder>label_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>966</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionChange_Path">
   <property name="text">
    <string>Change Path</string>
   </property>
  </action>
  <action name="actionOpenLog">
   <property name="text">
    <string>openLog</string>
   </property>
  </action>
  <action name="actionChangeExtension">
   <property name="text">
    <string>Change Extension</string>
   </property>
   <property name="toolTip">
    <string>Change output extension</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnPlay</sender>
   <signal>clicked()</signal>
   <receiver>actionPlay</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRecord</sender>
   <signal>clicked()</signal>
   <receiver>actionRecord</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnChangeOutput</sender>
   <signal>clicked()</signal>
   <receiver>actionChange_Path</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOpenLog</sender>
   <signal>clicked()</signal>
   <receiver>actionOpenLog</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmbExtension</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>actionChangeExtension</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
